config/presets/identity/email-password-interaction-policy.json

```
@@ -0,0 +1,216 @@
+{
+  "@context": "https://linkedsoftwaredependencies.org/bundles/npm/@solid/community-server/^0.0.0/components/context.jsonld",
+  "@graph": [
+    {
+      "@id": "urn:solid-server:email-password-interaction:StorageAdapter",
+      "@type": "EmailPasswordResourceStoreStorageAdapter",
+      "EmailPasswordResourceStoreStorageAdapter:_args_baseUrl": {
+        "@id": "urn:solid-server:default:variable:baseUrl"
+      },
+      "EmailPasswordResourceStoreStorageAdapter:_args_storagePathname": "/idp/email-password-db",
+      "EmailPasswordResourceStoreStorageAdapter:_args_saltRounds": 10,
+      "EmailPasswordResourceStoreStorageAdapter:_args_store": {
+        "@id": "urn:solid-server:default:ResourceStore"
+      }
+    },
+    {
+      "@id": "urn:solid-server:email-password-interaction:OidcInteractionCompleter",
+      "@type": "OidcInteractionCompleter"
+    },
+    {
+      "@id": "urn:solid-server:email-password-interaction:LoginRenderHandler",
+      "@type": "RenderEjsHandler",
+      "RenderEjsHandler:_viewsFolder": {
+        "@id": "urn:solid-server:default:variable:viewsFolder"
+      },
+      "RenderEjsHandler:_ejsTemplatePath": "./email-password-interaction/login.ejs"
```

Naming seems inconsistent. Suggestion:

```suggestion
      "RenderEjsHandler:_templatePath": {
        "@id": "urn:solid-server:default:variable:viewsFolder"
      },
      "RenderEjsHandler:_templateFile": "./email-password-interaction/login.ejs"
```

---

Done